servicetalk-transport-api/src/main/java/io/servicetalk/transport/api/SslConfig.java

```diff
@@ -155,9 +158,22 @@ public interface SslConfig {
      *
      * @return the list of certificate compression algorithms to advertise.
      * @see <a href="https://www.rfc-editor.org/rfc/rfc8879">RFC8879 - TLS Certificate Compression</a>
+     * @see CertificateCompressionAlgorithms
      */
     @Nullable // FIXME 0.43 - remove default implementation
     default List<CertificateCompressionAlgorithm> certificateCompressionAlgorithms() {
         return null;
     }
+
+    /**
+     * Get the timeout for the handshake process.
+     * <p>
+     * Implementations can round the returned {@link Duration} to full time units, depending on their time granularity.
+     * {@link Duration#ZERO Zero duration} disables the timeout.
+     *
+     * @return the timeout for the handshake process or {@link Duration#ZERO} to disable it.
```

Comment:
consider @see to the DEFAULT_HANDSHAKE_TIMEOUT

Reply:
I can not, it's pkg-private constant.
Until 1.0 we made a decision to avoid clarifying default values in javadoc to give us some flexibility to adjust them. We can reconsider after 1.0